iquip/apps/builder.py

```diff
@@ -459,17 +494,21 @@ def initArgsEntry(self, experimentInfo: ExperimentInfo):
         for argName, (argInfo, *_) in experimentInfo.arginfo.items():
             # TODO(BECATRUE): The other types such as 'Scannable'
             # will be implemented in Basic Runner project.
+            argType = argInfo.pop("ty")
             entryCls = {
                 "BooleanValue": _BooleanEntry,
                 "StringValue": _StringEntry,
                 "EnumerationValue": _EnumerationEntry,
-                "NumberValue": _NumberEntry
-            }[argInfo.pop("ty")]
+                "NumberValue": _NumberEntry,
+                "Scannable": _ScanEntry
+            }[argType]
             widget = entryCls(argName, argInfo)
-            item = QListWidgetItem(self.builderFrame.argsListWidget)
+            listWidget = (self.builderFrame.scanListWidget if argType == "Scannable"
+                          else self.builderFrame.argsListWidget)
```

Comment:
Why did you use parentheses?

Reply:
Otherwise, I encounter with error message.

Reply:
And the reference you gave me ([ref](https://google.github.io/styleguide/pyguide.html#32-line-length)) tells that using parentheses for line breaking.

Reply:
Oh, you're right👍